packages/autocomplete-theme-classic/src/theme.scss

```
@@ -1,229 +1,412 @@
+    --icon-color: rgb(119, 119, 163);
+    --text-color: rgb(38, 38, 39);
+    --background-color: rgb(255, 255, 255);
+    --panel-shadow: 0 0 0 1px rgba(35, 38, 59, 0.1), 0 6px 16px -4px rgba(35, 38, 59, 0.15);
 }
 
-.aa-InputWrapperPrefix,
-.aa-InputWrapperSuffix {
-  height: 100%;
-  position: absolute;
-  top: 0;
-  z-index: 1;
-}
-
-.aa-InputWrapperSuffix {
-  right: 0;
-}
-
-.aa-Label,
-.aa-TouchSearchButtonIcon {
-  align-items: center;
-  color: #777;
-  cursor: initial;
-  display: flex;
-  height: 100%;
-}
-
-.aa-InputWrapper,
-.aa-TouchSearchButton {
-  align-items: center;
-  background-color: #fff;
-  display: flex;
-  height: 2.5rem;
-  position: relative;
-  width: 100%;
-}
-
-.aa-Input,
-.aa-TouchSearchButton {
-  border: 1px solid #d6d6e7;
-  border-radius: 3px;
-  box-shadow: rgb(119 122 175 / 30%) 0 1px 4px 0 inset;
-  font: inherit;
-}
-
-.aa-TouchSearchButton {
-  color: #5a5e9a;
-  cursor: pointer;
-  padding: 0;
-  text-align: left;
-}
-
-.aa-Input {
-  appearance: none;
-  background: none;
-  caret-color: #5a5e9a;
-  color: #23263b;
-  height: 100%;
-  padding: 0 2.25rem;
-  position: absolute;
-  width: 100%;
-
-  &::-webkit-search-decoration,
-  &::-webkit-search-cancel-button,
-  &::-webkit-search-results-button,
-  &::-webkit-search-results-decoration {
-    appearance: none;
-  }
-
-  &::placeholder {
-    color: #5a5e9a;
-  }
-
-  &:focus {
-    border-color: #3c4fe0;
-    box-shadow: rgb(35 38 59 / 5%) 0 1px 0 0;
-    outline: currentColor none medium;
-  }
-}
-
-.aa-Label[hidden],
-.aa-LoadingIndicator[hidden] {
-  display: none;
-}
-
-.aa-SubmitButton,
-.aa-ResetButton,
-.aa-LoadingIndicator,
-.aa-TouchSearchButtonIcon {
-  height: 100%;
-  padding: 0 0.5rem;
-}
-
-.aa-SubmitButton,
-.aa-ResetButton {
-  background: none;
-  border: 0;
-  color: inherit;
-  cursor: pointer;
-}
-
-.aa-LoadingIndicator {
-  position: absolute;
-  z-index: 2;
-}
-
-.aa-LoadingIcon {
-  height: 100%;
-}
-
-.aa-List {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
-
-.aa-Item {
-  color: #23263b;
-  cursor: pointer;
-  display: flex;
-  grid-gap: 0.5rem;
-  justify-content: space-between;
-
-  mark {
-    background: none;
-    font-style: normal;
-    font-weight: bold;
-  }
-
-  /* stylelint-disable-next-line */
-  &[aria-selected='true'] {
-    background-color: #f5f5fa;
-  }
-}
-
-.aa-ItemLink {
-  color: inherit;
-  text-decoration: none;
-}
-
-.aa-ItemContent {
-  display: flex;
-  flex-grow: 1;
-  grid-gap: 0.5rem;
-  padding: 0.5rem;
-}
-
-.aa-ItemSourceIcon,
-.aa-ItemActionButton {
-  color: rgba(0, 0, 0, 0.4);
-}
-
-.aa-ItemActionButton {
-  background-color: transparent;
-  border: 0;
-  cursor: pointer;
-  opacity: 0.8;
-  padding: 0.5rem;
-  transition: color 100ms;
-  @media screen and (prefers-reduced-motion: reduce) {
-    transition: none;
-  }
-
-  &:hover,
-  &:focus {
-    color: rgba(80, 80, 80, 1);
-  }
+// ----------------
+// Darkmode
+// ----------------
+body {
+    &[data-theme='dark'],
+    &.dark {
```

```suggestion
.aa-Form {
    [data-theme='dark'] &,
    .dark & {
```

(whatever the root is)

---

what about the media query? https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme